inference-engine/ie_bridges/python/wheel/setup.py

```diff
@@ -120,6 +121,16 @@ class CustomBuild(build):
     def run(self):
         self.run_command('build_clib')
         build.run(self)
+        # Copy extra package_data content filtered by find_packages
+        dst = Path(self.build_lib)
+        src = Path(get_package_dir(PY_INSTALL_CFG))
+        exclude = ignore_patterns('*ez_setup*', '*__pycache__*', '*.egg-info*')
+        for path in src.glob('**/*'):
+            if path.is_dir() or exclude(str(path)):
+                continue
+            path_rel = path.relative_to(src)
+            (dst / path_rel.parent).mkdir(exist_ok=True, parents=True)
+            copyfile(path, dst / path_rel)
```

Comment:
could you please explain what happens here and why this file is somehow related to openvino-dev while it's runtime only?

Reply:
discussed offline